### DISCUSSION TITLE
Support for postgres/mariadb?

### DISCUSSION THREAD

**Discussion by goose-ws** [gilesknap/gphotos-sync#465] [Ideas]
First up is a big thank you to a tool I appreciate being able to use to keep all my things backed up.

Is there any thoughts or plans for support for postgres/mariadb over sqlite in the future? After a few bad experiences with sqlite once it starts to get a little top-heavy I'm trying to migrate away from it, as much as possible. Currently I'm at about 30k items in my Photos library, with no plans to stop using Photos in the future, so I'd appreciate a more durable database to power the tool on the back end.

**Comment by gilesknap** [comment]
Hi @goose-ws that's an interesting idea. I'm afraid I don't have enough commitment to this project at present to do anything like that. I would be happy to review PRs in this area.

If Google ever get around to fixing the API such that we are making true backups with this project then I'd get engaged again, maybe even do a rewrite in Rust. But I've given up any hope that this will happen several years ago.
